src/modules/account/accountState.js

```diff
@@ -483,6 +483,20 @@ class AccountState {
     return newWalletInfo.id
   }
 
+  listSplittableWalletTypes (walletId) {
+    const walletInfo = this.allKeys.find(
+      walletInfo => walletInfo.id === walletId
+    )
+    if (!walletInfo) throw new Error(`Invalid wallet id ${walletInfo.id}`)
+
+    // This is the bare minimum for now:
+    if (walletInfo.type === 'wallet:bitcoin') {
```

Comment:
Probably need something like ` && walletInfo.keys.type !== 'bip49'`

Reply:
Do you have access to the pluginCurrency info here?

Reply:
I believe we can get access to the currencyInfo from there, yes. It should be `this.ai.props.state.currency.infos`. I think there is also a selector, `getCurrencyInfo`, so you could do `getCurrencyInfo(this.ai.props.state, walletType)`

Reply:
If you access to the currencyInfos then we can do something like this:

```javascript
return pluginCurrencyInfoToSplitFrom.forks.filter(walletType => {
   pluginCurrencyInfoToSplitTo.formats.find(walletInfo.keys.format)
})
```

And I'll add 'formats' and 'forks' into the currencyInfos